platform/windows/os_windows.cpp

```diff
@@ -1204,7 +1204,14 @@ Error OS_Windows::initialize(const VideoMode &p_desired, int p_video_driver, int
 
 	AdjustWindowRectEx(&WindowRect, dwStyle, FALSE, dwExStyle);
 
+	char *windowid;
+#ifdef MINGW_ENABLED
```

Comment:
```diff
diff --git a/platform/windows/os_windows.cpp b/platform/windows/os_windows.cpp
index 432e694e1..ca6c8814f 100644
--- a/platform/windows/os_windows.cpp
+++ b/platform/windows/os_windows.cpp
@@ -1206,7 +1206,7 @@ Error OS_Windows::initialize(const VideoMode &p_desired, int p_video_driver, int
 
        char *windowid;
 #ifdef MINGW_ENABLED
-       char *windowid = getenv("GODOT_WINDOWID");
+       windowid = getenv("GODOT_WINDOWID");
 #else
        size_t len;
        _dupenv_s(&windowid, &len, "GODOT_WINDOWID");
```

Reply:
Bump, this is still needed to build with MINGW.